packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -500,6 +505,10 @@ Your plugin code is in $relativePluginMain.
 
 Host platform code is in the $platformsString directories under $relativePluginPath.
 To edit platform code in an IDE see https://flutter.dev/developing-packages/#edit-plugin-package.
+
+If you are adding new platforms to an existing plugin, you might need to update the pubspec.yaml file.
+Scroll up to find a detailed instruction on how to update the pubspec.yaml.
```

Comment:
> Scroll up to find a detailed instruction on how to update the pubspec.yaml

Maybe something more specific like "at the start of this message?"

If they didn't see the message before, perhaps we should link to some flutter.dev documentation on this topic instead